### PR TITLE
Add com.francescogaglione.chronos

### PR DESCRIPTION
**Pull request by francesco-gaglione**
### Please confirm your submission meets all the criteria

- [X] Please describe the application briefly.
  Chronos is a simple and intuitive Pomodoro timer for Cosmic desktop

- [X] The domain used for the application ID is [controlled by the application developers][appid-domain] and the [application id guidelines][appid] are followed.
- [X] I have read the and followed all the [Submission and licence requirements][reqs].
- [X] I have [built][build] and tested the submission locally.
- [X] I am an author/developer/upstream contributor of the project. If not, I contacted upstream developers about this submission. **Link:**

[appid-domain]: https://docs.flathub.org/docs/for-app-authors/requirements/#control-over-domain-or-repository
[appid]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id
[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission

### CODE REVIEW

File: com.francescogaglione.chronos.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        "cargo --offline build --release --verbose",
+        "install -Dm0755 ./target/release/chronos /app/bin/chronos",
+        "install -Dm0644 ./res/Chronos.desktop /app/share/applications/com.francescogaglione.chronos.desktop",
+        "install -Dm0644 ./res/Chronos.metainfo.xml /app/share/metainfo/com.francescogaglione.chronos.metainfo.xml",
+        "install -Dm0644 ./res/icons/hicolor/scalable/apps/com.francescogaglione.chronos.svg /app/share/icons/hicolor/scalable/apps/com.francescogaglione.chronos.svg"
+      ],
+      "sources": [
+        {
+          "type": "dir",
+          "path": "."
```
Comment (bbhtt) [inline com.francescogaglione.chronos.json:40]: ci isn't going to work with dir, it's on your PC. use `url` and type git.

File: com.francescogaglione.chronos.json [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  "sdk-extensions": ["org.freedesktop.Sdk.Extension.rust-stable"],
+  "command": "chronos",
+  "finish-args": [
+    "--share=ipc",
+    "--share=network",
+    "--socket=fallback-x11",
+    "--socket=wayland",
+    "--device=dri",
+    "--talk-name=com.system76.CosmicSettingsDaemon",
+    "--filesystem=xdg-config/cosmic:ro"
```
Comment (bbhtt) [inline com.francescogaglione.chronos.json:15]: you have to ask an exception for this.

Comment (bbhtt) [reply com.francescogaglione.chronos.json:15]: error code is `finish-args-unnecessary-xdg-config-cosmic-ro-access`

Comment (francesco-gaglione) [reply com.francescogaglione.chronos.json:15]: @bbhtt 
https://github.com/flathub-infra/flatpak-builder-lint/pull/490
Done it

**Comment by bbhtt** [comment]
bot, build com.francescogaglione.chronos

**Comment by francesco-gaglione** [comment]
@barthalion @bbhtt I accepted the repo invitation, but the app still not is on fluthub. There is some delay or I need to do something?

**Comment by bbhtt** [comment]
> @barthalion @bbhtt I accepted the repo invitation, but the app still not is on fluthub. There is some delay or I need to do something?

https://docs.flathub.org/docs/for-app-authors/submission#how-long-does-it-take-to-get-the-build-published-after-merge

**Pull request closed without merging**